bin/templates/cordova/lib/build.js

```diff
@@ -402,6 +440,16 @@ function parseOpts(options, resolvedTarget) {
                 case 'gradleArg':
                     ret.extraArgs.push(flagValue);
                     break;
+                case 'keystore':
+                case 'alias':
+                case 'keytorePassword':
+                case 'password':
+                case 'keystoreType':
+                    packageArgs[flagName] = flagValue;
+                    break;
+                case 'buildConfig':
+                    buildConfig = flagValue.replace(/"/g, '');
```

Comment:
My previous statement is not true. I think I found my issue where I was explicitly adding quotes in cordova-lib code which was escaped by superspawn. I do not need this and will remove it. Thanks!